<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Админ панель</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(216, 255, 236);</string>
  </property>
  <widget class="QTextEdit" name="Text">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>381</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>262</width>
     <height>482</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButtonAdd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonEdit">
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>80</height>
       </size>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonDelete">
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>80</height>
       </size>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton1">
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>80</height>
       </size>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton2">
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>80</height>
       </size>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton3">
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="Search">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAlphabetically</enum>
   </property>
   <item>
    <property name="text">
     <string>id</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>name</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>year</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>address</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="Search_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAlphabetically</enum>
   </property>
   <item>
    <property name="text">
     <string>id</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>name</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>year</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>address</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
